dashboard/app/models/student_work_evaluation.rb

```diff
@@ -0,0 +1,33 @@
+# == Schema Information
+#
+# Table name: student_work_evaluations
+#
+#  id                  :bigint           not null, primary key
+#  type                :string(255)
```

Comment:
assuming you do not plan to ever instantiate the parent class, I would recommend making this column "not null".
 
Even though this is somewhat redundant with the rails validation below, a rule at the database level gives you a stronger guarantee, which can be helpful for keeping the data in a good state. If you find you need to instantiate the parent class for some unforeseen reason, you could always relax this requirement at that time.